runner/runs/mujoco_halfcheetah_grid_search.py

```diff
@@ -0,0 +1,60 @@
+from runner.run_description import RunDescription, Experiment, ParamGrid
+
+_params = ParamGrid([
```

Comment:
This does not search over any seeds. To actually try multiple seeds you should add e.g. 'seed': [1,2,3,4] to the ParamGrid

Reply:
This file just like an example, and I make our parameter to be the same as rllib ppo, besides ppo_clip_ratio, since I found to set it as 4 is better than set it as 0.2.

Reply:
Any way, i will do some changes to make it looks like grid search